sdk/compiler/damlc/daml-doc/src/DA/Daml/Doc/Transform/Annotations.hs

```diff
@@ -84,14 +139,19 @@ applyHide = concatMap onModule
             | ADTDoc{..} <- x, all (isHide . ac_descr) ad_constrs = pure x{ad_constrs = []}
             | otherwise = [x]
 
-
-getAnn :: Maybe DocText -> [T.Text]
-getAnn = maybe [] (T.words . unDocText)
+-- Returns first line words and rest of lines
+getAnn :: Maybe DocText -> ([T.Text], [T.Text])
```

Comment:
Maybe a better name than `getAnn` would be good here, since at first I thought you might be extracting actual `AnnD` - how about `getCommentHeads` at first thought?